t/kubernetes/discovery/kubernetes3.t

```diff
@@ -381,6 +381,8 @@ POST /operators
 Content-type: application/json
 --- response_body_like
 .*"name":"default/kubernetes".*
+--- error-log
+attempt to get length of local 'endpointslices' (a userdata value)
```

Comment:
I don't quite understand why this assertion needs to be added here.

Reply:
It's been around a year now since I raised this PR. I also don't remember the exact reason I added the error log. I have removed it for now and tested it locally ..test if passing.